### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by alonpeled44**

### CODE REVIEW

File: components/LinkCard.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import styles from "@/styles/components/link-card.module.scss";
+
+export default function LinkCard({ name, url, icon }) {
+	return (
+		<a className={styles["link-card"]} href={`#${url}`}>
```
Comment (liogiladi) [inline components/LinkCard.jsx:5]: wrote it in a previous comment, I missed it. make the href usable for future cases. e.g: ```href={`/${url}`}```

File: components/Icons/AsksIcon.jsx
Comment (liogiladi) [inline components/Icons/AsksIcon.jsx:1]: make folder name in lower case

File: components/LinkCard.jsx (hunk 1 above)
Comment (alonpeled44) [reply components/LinkCard.jsx]: noted

**Review by liogiladi — APPROVED** [review]
nice

**Pull request merged**